transform_field/transform.py

```diff
@@ -80,6 +80,13 @@ def do_transform(record, field, trans_type, when=None):
             elif trans_type == "MASK-HIDDEN":
                 return_value = 'hidden'
 
+            # Transforms string input to masked version skipping first and last n characters
+            # e.g. MASK-STRING-SKIP-ENDS-3
+            elif 'MASK-STRING-SKIP-ENDS' in trans_type:
+                skip_ends_n = int(trans_type[-1])
+                return_value = '*'*len(value) if len(value) <= (2 * skip_ends_n) \
+                    else f'{value[:skip_ends_n]}{"*"*(len(value)-(2*skip_ends_n))}{value[-skip_ends_n:]}'
```

Comment:
there are several calls to `len`, can you call it once and save it to a variable  
```suggestion
                value_len = len(value)
                return_value = '*'*value_len if value_len <= (2 * skip_ends_n) \
                    else f'{value[:skip_ends_n]}{"*"*(value_len-(2*skip_ends_n))}{value[-skip_ends_n:]}'
```

Reply:
Done